### ISSUE TITLE
histology and Histopathology

### ISSUE THREAD

**Issue by masciam** [obi-ontology/obi#853]
I am working on developing an ontology for histopathology reports. 
In OBI histology has as synonym  histopathology http://purl.obolibrary.org/obo/OBI_0600020
I think that these should be two different terms.
histology is as in OBI "The visual examination of cells or tissue (or images of them) with an assessment regarding a quality of the cells or tissue. Parts are: staining, imaging, judgement"
But since not always histology is used for pathological specimen such as in the case of the lab test to check the location of some expression molecules  I think that we should have two different term
With histopathology been an histology with the purpose of examine the pathological level of the specimen

**Comment by bpeters42** [comment]
I agree. We tend to use synonyms loosely, especially if no one asks for
separate terms. Can you submit a term request with definition etc?

**Comment by masciam** [comment]
Histopathology is child of histology 
proposed  definition: Histopathology is an histology that  has_specified_output some (information content entity and (is about some (disease)))
I think the difference between a general histology and the histopathology is the purpose the histopathology is performed to check the presence or the level of a specific disease.

**Comment by jamesaoverton** [comment]
Discussed on the OBI call 2017-07-31. We were agreed that 'histology' is more general and 'histopathology' is focused on pathology, and that it would be good to have distinct terms.

We could also make a link to ['pathologist role'](http://purl.obolibrary.org/obo/OBI_0000145).

For the textual definition of 'histopathology', we suggest: "A histology assay that is intended to check for the presence or level of a specific disease".

**Comment by mbrochhausen** [comment]
I did add the term (https://github.com/obi-ontology/obi/commit/30885d82a11c1dbf9950d7b97597e5847d4f4ee4), but I am going to keep the issue open, since there are two more things to fix, in my mind:
1. delete alternative term 'histopathology' from 'histology'
2. The definition know reads "A histology assay...". Either we should rename 'histology' to 'histology assay' (which I would suggest) and add 'histology' as an alternative term, or the definition should be changed to "A histology..."

**Comment by masciam** [comment]
Fine with me.
Thanks
AM

Sent from my iPhone

On Jul 31, 2017, at 12:39 PM, James A. Overton <notifications@github.com<mailto:notifications@github.com>> wrote:


Discussed on the OBI call 2017-07-31. We were agreed that 'histology' is more general and 'histopathology' is focused on pathology, and that it would be good to have distinct terms.

We could also make a link to 'pathologist role'<https://urldefense.proofpoint.com/v2/url?u=http-3A__purl.obolibrary.org_obo_OBI-5F0000145&d=DwMFaQ&c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&r=HyovC1qaIstHD1mU7n3_tykAUJmKYdES-1qpzSs7PCs&m=6nueMeu5kQ1tNPpTUBu3C5v_wLaoZAmsaBA_i4BspAY&s=sNQwUGsFlDyKKToUahCN-oslwHpK14kvUCA9binew2g&e=>.

For the textual definition of 'histopathology', we suggest: "A histology assay that is intended to check for the presence or level of a specific disease".

**Comment by masciam** [comment]
I am fine with both points that Mathias suggested .
AM


Sent from my iPhone

On Aug 4, 2017, at 1:00 PM, Mathias Brochhausen <notifications@github.com<mailto:notifications@github.com>> wrote:


I did add the term (30885d8<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_obi-2Dontology_obi_commit_30885d82a11c1dbf9950d7b97597e5847d4f4ee4&d=DwMFaQ&c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&r=HyovC1qaIstHD1mU7n3_tykAUJmKYdES-1qpzSs7PCs&m=doh72xRhhWYaT2gdg44WVoAvMbJ50A1ml-DJo4SUYuM&s=04N_4ValPEqdDW19qoWuCEF7p_wJEmDICPGHm7BNLgA&e=>), but I am going to keep the issue open, since there are two more things to fix, in my mind:

  1.  delete alternative term 'histopathology' from 'histology'
  2.  The definition know reads "A histology assay...". Either we should rename 'histology' to 'histology assay' (which I would suggest) and add 'histology' as an alternative term, or the definition should be changed to "A histology..."

**Comment by turbomam** [comment]
Inclined to add "assay" to the labels and then close.  Speak up  if it needs more attention.
